### ISSUE TITLE
Please consider rewriting these instructions...

### ISSUE THREAD

**Issue by knowlesy14** [microsoft/AADConnectConfigDocumenter#32]
Please consider rewriting these instructions. I found them quite hard to follow.

**Comment by NileshGhodekar** [comment]
HI @knowlesy14 , please submit a pull request with improved instructions and I'll review the updates. Thanks!

**Comment by dfw111** [comment]
Agreed. I've read this about 10 times and having difficulty with it. And I would like to contribute to a re-write. But I don't fully understand the purpose of this tool.

**Comment by dfw111** [comment]
Perhaps change out "Pilot / Test" and "Target / Pilot" with "New".
And change out "Reference / Production" and "Baseline / Production" with "Existing".

**Comment by dfw111** [comment]
Here's what I have so far: 
Extract the zip file to an empty local folder on a machine which has .NET Framework 4.5 installed. 
•	Example: C:\Temp\AzureADConnectSyncDocumenter
•	This will extract the Documenter files and sample data files for "Contoso".
•	Run AzureADConnectSyncDocumenter-Contoso.cmd and ensure no errors are reported in AADConnectSyncDocumenter-Error.log
•	This CMD doesn’t make any changes. It simply reads sample data from the Contoso directory and creates AADConnectSyncDocumenter.log

•	Export the Server Configuration of your new or existing Azure AD Connect server by running Get-ADSyncServerConfiguration cmdlet defined in ADSync module shipped with Azure AD Connect.
	Import-Module ADSync 
	Get-ADSyncServerConfiguration -Path "<CompletePathToOutputFolder>"
	Example: Get-ADSyncServerConfiguration -Path “C:\temp\AzureADConnectSyncDocumenter\Output”

**Comment by dfw111** [comment]
•	Go to C:\temp\AzureADConnectSyncDocumenter and create a new folder next to Contoso with your company name.
•	If you are upgrading your server to a new version of Azure AD Connect, then…..
•	This is where I get a bit lost. Do we upgrade AD Connect then rerun the Documenter.cmd? And then do a comparison with the Report.html?

**Comment by dfw111** [comment]
So then I did some word swaps, but it's kind of confusing at this point. 

•	Copy the configuration export files from the Output folder to a folder under the "Data" directory of the Documenter tool. 
o	e.g. the "New" configuration files for the customer "Contoso" are provided as a sample under the "Data\Contoso\New" folder. This is the folder where the files of the new configuration is located. So, for example, in a swing migration project the "New" configuration will be the configuration of the "old" server, since the goal is to build a new server with the configuration as close to the old server as possible.
•	If you want to document the changes from a specific baseline, export the server configuration of your existing Azure AD Connect server and copy the output to a folder under the Documenter "Data" directory. 
o	e.g. the "Existing" configuration files for the customer "Contoso" are provided as a sample under the "Data\Contoso\Existing" folder. This is the folder where the files of the existing configuration is located.
•	Make a copy of AzureADConnectSyncDocumenter-Contoso.cmd with a name appropriate for your environment (say AzureADConnectSyncDocumenter-Fabrikam.cmd) and edit the folder paths for "New" and "Existing" with your Fabrikam name. 
o	If you don't have a existing config, specify the same path as the "New" config.
•	Run your updated batch file. Upon successful execution, the generated report will be found in the Documenter "Report" folder.
•	View the HTML file and somehow a difference will be shown.

**Comment by theheatDK** [comment]
Thanks for taking the time to create this tool!
I gave up using it since I don't understand the documentation.
Does anybody know of alternative tools?
I just need a simple report of the configuration of an existing Azure AD Connect installation, nothing else
Currently, I use screenshots which is not great.
Regards Peter

**Comment by NileshGhodekar** [comment]
Your usecase  is simple to explain. The tool's executable expects two sets of input configuration files to be provided in it's "Data" folder, let's call them A and B. In your case B is same as A. So on the commandline you'll be running is: AzureADConnectSyncDocumenterCmd.exe "A" "A". That is all. Again, if your config files are placed in the folder "FABRIKAM\07JUL2021" then you'd run  AzureADConnectSyncDocumenterCmd.exe "FABRIKAM\07JUL2021" "FABRIKAM\07JUL2021"..

It's only when you have two different sets of config files and would be interested to understand how the changes are reported and in the PowerShell script  that it also produces for recreating sync rules that it becomes super important what you call as A and you call as B and it can get quite confusing which is A and which is B based on what you are exactly trying to do - promote the change from test to prod, staging to prod, prod to staging, swing migration, etc.. The terminology for naming A=Pilot/Target and B=Production/Baseline is carried over from FIM/MIM world.

**Comment by NileshGhodekar** [comment]
After much thought, I've just documented the instructions separately for four different main use cases. Hopefully this is clearer and helpful.

**Comment by NileshGhodekar** [comment]
Hi @texaskat, updates were on the wiki tab homepage, but now visible on the project homepage as well.
